js/form.js

```diff
@@ -10,30 +26,73 @@ function setFormEnabled (formName, disableClassName) {
 setFormDisabled(mapFiltersForm, 'ad-form--disabled');
-
 setFormEnabled(adForm, 'ad-form--disabled');
 
-// function changeFormAccess (formName, disableClassName) {
-//   if (formName.classList.contains(disableClassName)) {
-//     formName.classList.remove(disableClassName);
-//     [...formName.children].forEach((element) => {element.disabled = false;});
-//   }
-//   else {
-//     formName.classList.add(disableClassName);
-//     [...formName.children].forEach((element) => {element.disabled = true;});
-//   }
-// }
-
-// const promoBlock = document.querySelector('.promo');
-// promoBlock.style.flexDirection = 'column';
-// promoBlock.style.alignItems = 'center';
+const titleInput = document.querySelector('#title');
+titleInput.addEventListener('input', () => {
+  const valueLength = titleInput.value.length;
+  if (valueLength < MIN_NAME_LENGTH) {
+    titleInput.setCustomValidity(`Ещё ${ MIN_NAME_LENGTH - valueLength} символов`);
+  } else if (valueLength > MAX_NAME_LENGTH) {
+    titleInput.setCustomValidity(`Удалите лишние ${ MAX_NAME_LENGTH - valueLength} символов`);
+  } else {
+    titleInput.setCustomValidity('');
+  }
+  titleInput.reportValidity();
+});
 
-// const buttonElement = document.createElement('button');
-// buttonElement.innerHTML = 'Enable';
-// promoBlock.appendChild(buttonElement);
+const roomNumberInput = document.querySelector('#room_number');
+const apartmentCapacityInput = document.querySelector('#capacity');
+const apartmentTypeInput = document.querySelector('#type');
+const priceInput = document.querySelector('#price');
+const timeInInput = document.querySelector('#timein');
+const timeOutInput = document.querySelector('#timeout');
 
-// buttonElement.addEventListener('click', () => {changeFormAccess(mapFiltersForm, 'ad-form--disabled');});
+const isProperRelation = function (){
+  return relationRoomsGuests[roomNumberInput.value].includes(apartmentCapacityInput.value);
+};
+//соответствие числа гостей количеству комнат
+apartmentCapacityInput.addEventListener('change', () => {
+  if (!isProperRelation()) {apartmentCapacityInput.setCustomValidity('Недопустимое число гостей');}
+  else {apartmentCapacityInput.setCustomValidity('');}
+  apartmentCapacityInput.reportValidity();
+});
+//соответствие числа комнат количеству гостей
+roomNumberInput.addEventListener('change', () => {
+  if (!isProperRelation()) {roomNumberInput.setCustomValidity('Несоответствие: комнаты-гости!!');}
+  else {roomNumberInput.setCustomValidity('');}
+  roomNumberInput.reportValidity();
+});
+//добавляем атрибуты в поле ввода цены
+priceInput.min = relationTypeMinPrice[apartmentTypeInput.value];
+priceInput.placeholder = relationTypeMinPrice[apartmentTypeInput.value];
+const isProperPrice = function () {
+  if (parseInt(priceInput.value, 10) < parseInt(priceInput.min, 10)) {
+    priceInput.setCustomValidity(`Цена должна быть выше ${priceInput.min}`);
+  } else {priceInput.setCustomValidity('');}
```

Comment:
понимаю что линтер так пропускает, но лучше, разносить это на отдельные строки
```suggestion
  } else {
    priceInput.setCustomValidity('');
  }
```